rspirv/sr/storage.rs

```diff
@@ -16,10 +16,7 @@ pub struct Token<T> {
 
 impl<T> Clone for Token<T> {
     fn clone(&self) -> Self {
-        Token {
-            index: self.index,
-            marker: self.marker,
-        }
+        *self
```

Comment:
Does this now call `Copy`... which calls `Clone`?

Reply:
`Copy` doesn't call `Clone` it's a memcpy internally, there is an example of this pattern in the rustdocs as well
https://doc.rust-lang.org/std/marker/trait.Copy.html#how-can-i-implement-copy